# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1256774435 14400
#      Wed Oct 28 20:00:35 2009 -0400
# Node ID c6b97d21834586a2948f86e830dec67315266cbc
# Parent  97de6c916498bb42d7e04a5f135c27eeb6440e00
PermMatrix.cc: don't include error.h

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,12 @@
+2009-10-28  John W. Eaton  <jwe@octave.org>
+
+	* PermMatrix.cc: Don't include error.h.
+
 2009-10-27  David Bateman  <dbateman@free.fr>
 
 	* SparsedbleLU.cc (SparseLU::SparseLU (const SparseMatrix&,
 	const Matrix&, bool), SparseLU::SparseLU (const SparseMatrix&,
 	const ColumnVector&, const Matrix&, boo, bool, double, bool,
 	bool)): Fix test for valid pivot tolerance.
 	* SparseCmplxLU.cc (SparseComplexLU::SparseComplexLU 
 	(const SparseComplexMatrix&, const Matrix&, bool), 
diff --git a/liboctave/PermMatrix.cc b/liboctave/PermMatrix.cc
--- a/liboctave/PermMatrix.cc
+++ b/liboctave/PermMatrix.cc
@@ -21,17 +21,16 @@ along with Octave; see the file COPYING.
 */
 
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include "PermMatrix.h"
 #include "idx-vector.h"
-#include "error.h"
 #include "Array-util.h"
 #include "oct-locbuf.h"
 
 static void
 gripe_invalid_permutation (void)
 {
   (*current_liboctave_error_handler)
     ("PermMatrix: invalid permutation vector");
